packages/danger/src/Utils.ts

```diff
@@ -12,7 +12,7 @@ export function getPackageChangelogRelativePath(packageName: string): string {
 
 export function getExpoRepositoryRootDir(): string {
   // EXPO_ROOT_DIR is set locally by direnv
-  return process.env.EXPO_ROOT_DIR || join(__dirname, '..');
+  return process.env.EXPO_ROOT_DIR || join(__dirname, '../../..');
```

Comment:
Why the path is different than you expected? 🤔

Reply:
I moved `danger` to the `packages` and I didn't change this path. 
Locally, we have set `EXPO_ROOT_DIR`. So, I didn't catch this problem earlier.